examples/index.html

```diff
@@ -39,6 +39,9 @@ <h3 id=html>Custom Elements</h3>
     <a title=&lt;input-output&gt; href=/examples/input-output.html>&lt;input-output&gt;</a>
 
   <li>
+    <a title=&lt;multiple-choice&gt; href=/examples/multiple-choice>&lt;multiple-choice&gt;</a>
```

Comment:
(which tbh is probably the reason why quotes are generally used by default, so that it can be 100% consistent in all cases)